src/platform/platform_posix.c

```diff
@@ -159,14 +161,33 @@ CxPlatInitialize(
     void
     )
 {
+    QUIC_STATUS Status;
+
     RandomFd = open("/dev/urandom", O_RDONLY|O_CLOEXEC);
     if (RandomFd == -1) {
-        return (QUIC_STATUS)errno;
+        Status = (QUIC_STATUS)errno;
+        goto Exit;
+    }
+
+#ifdef CX_PLATFORM_LINUX
+    struct sysinfo MemInfo;
+    if (sysinfo(&MemInfo) != 0) {
```

Comment:
Think we should log failures for this or `RandomFd` above?

Reply:
Probably. I can add one.